servicecontrol/servicecontrol-instances/configuration.md

```diff
@@ -754,6 +754,23 @@ Specifies any additional databases on the same server that also contain NService
 | --- | --- |
 | string | |
```

Comment:
i noticed that this one is blank; do you know if that is intentional?

Reply:
because the default value is empty/nothing